### PR TITLE
Fixes #14032 - Ostree branches now searchable in a Repo

### PR DESCRIPTION
**Pull request by parthaa**

### CODE REVIEW

**Comment by ehelms** [comment]
Looks good -- tests?

**Comment by parthaa** [comment]
@ehelms hmm dont have a synced  ostree repo where I could test the real thing . Jeff mentioned there was a way I can create a real small repo. I 'll see if i can do that tomorrow..

**Comment by parthaa** [comment]
@ehelms updated

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 Exposing Pulp's juiciest parts. http://www.pulpproject.org/
 
-Latest Live Tested Version: **pulp-server-2.8.0-0.4.beta.el7.noarch**
+Latest Live Tested Version: **pulp-server-2.8.0-0.5.beta.el7.noarch**
```
Comment (ehelms) [inline README.md]: Shouldn't this be ` pulp-server-2.8.0-0.7.beta.el7.noarch` now?

**Comment by parthaa** [comment]
@ehelms updated

**Comment by ehelms** [comment]
ACK

**Pull request merged**